pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -220,6 +220,100 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: NodeRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     util.NewStringSet("list", "watch"),
+					Resources: util.NewStringSet("services"),
+				},
+
+				{
+					Verbs:     util.NewStringSet("get", "list", "watch"),
+					Resources: util.NewStringSet("nodes"),
+				},
+				{
+					// TODO: restrict to the bound node once supported
+					Verbs:     util.NewStringSet("update"),
+					Resources: util.NewStringSet("nodes/status"),
+				},
+
+				{
+					// TODO: restrict to pods scheduled on the bound node once supported
+					Verbs:     util.NewStringSet("list", "watch"),
+					Resources: util.NewStringSet("pods"),
+				},
+				{
+					// TODO: remove once mirror pods are removed
+					// TODO: restrict deletion to mirror pods created by the bound node once supported
+					// Needed for the node to create/delete mirror pods
+					Verbs:     util.NewStringSet("get", "create", "delete"),
+					Resources: util.NewStringSet("pods"),
+				},
+				{
+					// TODO: restrict to pods scheduled on the bound node once supported
+					Verbs:     util.NewStringSet("update"),
+					Resources: util.NewStringSet("pods/status"),
+				},
+
+				{
+					// TODO: restrict to secrets used by pods scheduled on bound node once supported
+					// Needed for imagepullsecrets and secret volumes
+					Verbs:     util.NewStringSet("get"),
+					Resources: util.NewStringSet("secrets"),
+				},
+
+				{
+					// TODO: restrict to claims/volumes used by pods scheduled on bound node once supported
+					// Needed for persistent volumes
+					Verbs:     util.NewStringSet("get"),
+					Resources: util.NewStringSet("persistentvolumeclaims", "persistentvolumes"),
+				},
+			},
+		},
+
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: SDNReaderRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     util.NewStringSet("get", "list", "watch"),
+					Resources: util.NewStringSet("hostsubnets"),
+				},
+				{
+					Verbs:     util.NewStringSet("list", "watch"),
+					Resources: util.NewStringSet("nodes"),
+				},
+				{
+					Verbs:     util.NewStringSet("get"),
+					Resources: util.NewStringSet("clusternetworks"),
+				},
+			},
+		},
+
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: SDNManagerRoleName,
```

Comment:
@rajatchopra can you check that this covers the read/write permission required by the sdn manager?